### PR TITLE
E-mail add support for sending via sendmail

### PR DESCRIPTION
**Pull request by mavimo**
This PR:
 * Remove dependency from swiftmail for the email plugin.
 * Introduce a mail template also for successful build.
 * Code cleanup removing unrequired dependencies.
 * CTest refactoring to remove dependencies from Swiftmail and having a isolated component to test (mail options, status code, subject, body, ..).

refs #598 cc/ @dancryer @tvbeek

### CODE REVIEW

File: PHPCI/Plugin/Email.php [hunk 1]
```diff
@@ -52,25 +48,16 @@ class Email implements \PHPCI\Plugin
...
-                           ? $phpCiSettings['email_settings']['from_address']
-                           : "notifications-ci@phptesting.org";
-
-        $this->mailer = $mailer;
     }
 
     /**
-    * Connects to MySQL and runs a specified set of queries.
-    */
+     * Send a notificaiton mail.
```
Comment (thaJeztah) [inline PHPCI/Plugin/Email.php]: Typo; notificaiton -> notification

**Comment by dancryer** [comment]
@mavimo Please can you rebase this onto the current master branch? :)

**Comment by mavimo** [comment]
@dancryer merged, conflict resolved and uniformed tests.

**Comment by tvbeek** [comment]
@mavimo Can you rebase it again? Ping me if done and I will merge it a.s.a.p.

**Comment by tvbeek** [comment]
Thanks. This pull request has been integrated manually.

**Pull request closed without merging**